src/encoded/static/scss/encoded/modules/_status.scss

```diff
@@ -12,6 +12,7 @@ $status-colors: (
     exempt-from-standards: #fff712,
     archived: #a0a0a0,
     revoked: $brand-danger,
+    preferred-default: #004d00,
```

Comment:
Again, I wouldn’t corrupt the existing status system. Could you put a star in the file graph node instead of using a different color? I think it seems confusing to call “preferred default” a status when it’s not, and to use different symbology for the file graph and file table. You can add U+2605 for the star that you can pass to the file graph text.

Reply:
I added the the star to the file graph-node name.